lib/vagrant/plugin/v2/synced_folder.rb

```diff
@@ -54,6 +56,12 @@ def disable(machine, folders, opts)
         # @param [Hash] opts
         def cleanup(machine, opts)
         end
+
+        def _initialize(machine, synced_folder_type)
+          plugins = Vagrant.plugin("2").manager.synced_folders
+          capabilities = Vagrant.plugin("2").manager.synced_folder_capabilities
+          initialize_capabilities!(synced_folder_type, plugins, capabilities, machine)
```

Comment:
would be neat if this was defined by the plugin. Something like a constant `SYNCED_FOLDER_TYPE` and then this could be a `initialize_capabilities!(self::SYNCED_FOLDER_TYPE, plugins, capabilities, machine)`